srcpkgs/ImageMagick/template

```diff
@@ -34,6 +34,13 @@ if [ -z "$CROSS_BUILD" ]; then
 	subpackages+=" libmagick-perl"
 fi
 
+case "$XBPS_TARGET_MACHINE" in
+	x86_64-musl)
```

Comment:
doubt this is only relevant for x86_64, all archs on musl 1.2 should probably have it (i.e. musl and 64bit)